python/kwiver/sprokit/adapters/adapter_data_set.cxx

```diff
@@ -182,6 +219,8 @@ PYBIND11_MODULE(adapter_data_set, m)
     // First the native C++ types
     .def("_add_int", &ka::adapter_data_set::add_value<int>)
     .def("_add_float", &ka::adapter_data_set::add_value<float>)
+    .def("_add_double", &ka::adapter_data_set::add_value<double>)
```

Comment:
Usually, `_` before a function call is interpreted as a private function in python. This implies that it shouldn't be called outside the scope of the class. Just want to clarify if that is the intent here?

Reply:
Yes, that is the intent here